tests/fixtures/kamereon/Contributions.rst

```diff
@@ -0,0 +1,27 @@
+Contributing
```

Comment:
CONTRIBUTING file

Reply:
I have created PR #71 to address this comment.

But where do you recommend that we add comments about the json files that people have already provided?